pytest_doctestplus/plugin.py

```diff
@@ -203,10 +210,10 @@ def collect(self):
                         self.fspath)
             else:
                 try:
-                    module = self.fspath.pyimport()
+                    module = fspath.pyimport()
```

Comment:
This change broke something.

Reply:
Yeah, that method doesn't exist on `Path` objects, so I will need to reimplement it I guess.